devices/xiaomi.js

```diff
@@ -1381,6 +1381,26 @@ module.exports = [
         },
         ota: ota.zigbeeOTA,
     },
+    {
+        zigbeeModel: ['lumi.curtain.acn003'],
+        model: 'ZNCLBL01LM',
+        vendor: 'Xiaomi',
+        description: 'Aqara curtain driver E1',
+        fromZigbee: [fz.battery, fz.xiaomi_curtain_position_tilt, fz.aqara_opple],
+        toZigbee: [tz.xiaomi_curtain_position_state, tz.ZNCLBL01LM],
```

Comment:
Accordingly, `tz.ZNCLBL01LM` must be replaced by `tz.ZNCLBL01LM_battery_voltage` and `tz.ZNCLBL01LM_hooks_state`.